tests/video/test_video.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim: ai ts=4 sts=4 et sw=4 nu
+
+import pathlib
+import subprocess
+import humanfriendly
+import shutil
+import pytest
+
+from zimscraperlib.download import save_large_file
+
+
+def get_video_info(src_path):
```

Comment:
Should we move that to the library? It seems quite useful although we have not been using such info so far